dyn_em/module_diffusion_em.F

```diff
@@ -8099,13 +8099,7 @@ SUBROUTINE vertical_diffusion_implicit(ru_tendf, rv_tendf, rw_tendf, rt_tendf,&
     ENDDO
  
   CASE (1) ! use surface heat flux computed from surface routine
-    DO j = j_start, j_end
-    DO i = i_start, i_end
-       cpm = cp * (1. + 0.8 * moist(i,kts,j,P_QV))
-       heat_flux = hfx(i,j)/cpm/rho(i,1,j)
-    ENDDO
-    ENDDO
- 
+    ! do nothing
```

Comment:
@dudhia 
Jimy,
We need you to evaluate this change.

Reply:
correct: neither heat_flux nor cpm are needed below. This will have no effect.